packagelist/debian_12_x11.txt

```diff
@@ -75,11 +75,22 @@ build-essential
 command-not-found
 gdb
 git
+libcairo2-dev
+libgif-dev
 libiio-dev
+libjpeg-dev
+libpng-dev
 libubootenv-tool
 lsof
 meson
 nfs-common
+xxd
+
+# Development tools - Python
+python3-bluez
+python3-can
+python3-libiio
+python3-pip
```

Comment:
If the size ends up being reasonable for including the python packages, we should make sure this matches the headless install.

Reply:
If we're exploring this, and `python-bluez` stays around and we're okay adding other convenience packages, I would also propose `https://github.com/ukBaz/python-bluezero` for consideration (I believe this also pulls in `python-aioblescan` which is also a great tool.

I propose this because this is what our BLE Examples documentation documents and uses https://docs.embeddedts.com/BLE_Examples#Example_BLE_Library_python-bluezero

It's completely fine left out, and I would be fine nixing all by python base and leaving the rest up to end implementation. But if its small and simple enough, it might be worth.

Reply:
From my recent adventures in python I'm leaning more towards adding pip by default and leave the rest to users setting up their own venv.

Reply:
With the recent adventures MPeters had with python and Debian, we should probably take a stance now on our platforms if we want to have pip handle everything (and if so, how to disable it if Debian wants to manage everything), or we we want APT to handle the python packages.

Reply:
Having `pip` for the system Python is destabilizing on a desktop or server system.

On an embedded board meant for the customer to experiment and test with and figure out how they can build it into exactly what they want, this isn't the same situation.

Reply:
`python-bluezero` sounds like an excellent suggestion, but I'd like to address it after `distro-seed` reaches parity with what we have already been shipping, e.g. on the 7100 and 7180. In particular, I don't see a pre-built Debian package for it, which would mean it isn't as simple as including it in these package lists.

Reply:
Lets use the existing Debian packages then and add python's gpio bindings. I'd consider this part resolved as long as we make sure we have these packages consistent between our debian/ubuntu images.

Reply:
The latest version of this PR has `python3-libgpiod` in all non-minimal package lists.